templates/individualModuleInfo.html

```diff
@@ -10,6 +10,160 @@
+				labelFontColor: "#161616",
+				lineColor: "#161616"
+			},
+			legend: {
+				verticalAlign: "bottom",
+				horizontalAlign: "center"
+			},
+			theme: "theme1",
+			data: [{
+						type: "column",
+						dataPoints: [      
+						$for area in focus_area_acronyms:
+							{y: $focus_area_counts[area], label: "$area"},
+						]
+				  }]
+		});
+
+		focusAreaBarChart.render();
+
+		focusAreaPieChart = new CanvasJS.Chart("module-view-focus-area-pie-chart",
+	    {
+			animationEnabled: true,
+			legend: {
+				verticalAlign: "center",
+				horizontalAlign: "left"
+			},
+			data: [{
+						type: "pie",
+						indexLabelFontFamily: "Garamond",       
+						indexLabelFontSize: 20,
+						indexLabelFontWeight: "bold",
+						startAngle:0,
+						indexLabelFontColor: "#161616",       
+						indexLabelLineColor: "#161616", 
+						indexLabelPlacement: "outside",
+
+						toolTipContent: "{name}: {y} (#percent%)",
+						showInLegend: false,
+						indexLabel: "{name}:  {y} (#percent%)", 
+						dataPoints: [
+						$for area, count in focus_area_counts.iteritems():
+							{y: $focus_area_counts[area], name: "$area", legendMarkerType: "square"},
+						]
+				  }]
+		});
+	}
+
+	function showStudentYearBarChart() {
+		document.getElementById('module-view-student-year-bar-chart').style.display = "block";
+		document.getElementById('module-view-student-year-pie-chart').style.display = "none";
+		document.getElementById('module-view-student-year-bar-chart-button').classList.add("active");
+		document.getElementById('module-view-student-year-pie-chart-button').classList.remove("active");
+		studentYearBarChart.render();
+	}
+
+	function showStudentYearPieChart() {
+		document.getElementById('module-view-student-year-pie-chart').style.display = "block";
+		document.getElementById('module-view-student-year-bar-chart').style.display = "none";
+		document.getElementById('module-view-student-year-pie-chart-button').classList.add("active");
+		document.getElementById('module-view-student-year-bar-chart-button').classList.remove("active");
+		studentYearPieChart.render();
+	}
+
+	function showFocusAreaBarChart() {
+		document.getElementById('module-view-focus-area-bar-chart').style.display = "block";
+		document.getElementById('module-view-focus-area-pie-chart').style.display = "none";
+		document.getElementById('module-view-focus-area-bar-chart-button').classList.add("active");
+		document.getElementById('module-view-focus-area-pie-chart-button').classList.remove("active");
+		focusAreaBarChart.render();
+	}
+
+	function showFocusAreaPieChart() {
+		document.getElementById('module-view-focus-area-pie-chart').style.display = "block";
+		document.getElementById('module-view-focus-area-bar-chart').style.display = "none";
+		document.getElementById('module-view-focus-area-pie-chart-button').classList.add("active");
+		document.getElementById('module-view-focus-area-bar-chart-button').classList.remove("active");
+		focusAreaPieChart.render();
+	}
+</script>
+
```

Comment:
Is it possible to use a JavaScript file instead of injecting the whole script into the HTML ?